tests/101-test-use-devs-to-create-thin-pool.sh

```diff
@@ -0,0 +1,45 @@
+source $SRCDIR/libtest.sh
+
+# Test DEVS= directive. Returns 0 on success and 1 on failure.
+test_devs() {
+  local devs=$TEST_DEVS
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local INFILE=storage-setup
+  local OUTFILE=storage
```

Comment:
So this will create files in current working directory of the test? We already create a directory called temp/ in cwd. How about putting these files there?

That way these will not spoil the code dir if I do tests/run-tests.sh